### ISSUE TITLE
Make custom dataset of other city alternative to SF-XL

### ISSUE THREAD

**Issue by songkemail** [gmberton/CosPlace#43]
Thanks for your great job, gmberton.
I'm wondering how to make my custom dataset that collected street view images of other city as well as the 6 DOFs, since the street images is 4096*2048 which is 2:1 ratio, and I seen the SF-XL is 3328*512 which is likely clipped from the origin 3328*1164 street images, should I clip the street images first, how much pixel should cut out from upper and bottom? 
But it will cut off the top of the buildings as well since there are many high buildings in my images, which same as SF-XL as well, how to do with that?
how about affixing the 2:1 panoramic image to a sphere , and then clip 512*512 tiles as datasets, which can avoid image distortion too?

**Comment by gmberton** [comment]
Hi, for SF-XL we had 3328 x 1664 panoramas. We removed the top 512 and bottom 640 pixels, resulting in 3328 x 512. I agree with you that this might not be the best option, and the best option is dependent on the area (as you say tall buildings might benefit from removing less pixels above the horizon), so I advise to try different approaches for removing pixels.
Then we create 12 partially overlapping crops of dimension 512 x 512. Again, in your case using less or more, bigger or smaller crops might help (we didn't tune these parameters).
The images are distorted, as we do not apply any undistortion. Years ago I tried to undistort some of them and results did not change much, so I kept them as they are.
Finally, the CosPlace code requires the image to have UTM east, UTM north, and YAW as fields of the filename. Make sure that when you create the crops, each crop has the correct YAW.
